### PR TITLE
Interactive msg optional title

### PR DESCRIPTION
**Pull request by AkhileshNegi**
- Added support for making title option while sending an interactive message
- Updated docs
- Updated APIs
- Added test cases

### CODE REVIEW

File: lib/glific/messages.ex [hunk 1]
```diff
@@ -246,7 +246,11 @@ defmodule Glific.Messages do
              interactive_template_id
            ),
          interactive_content <-
-           InteractiveTemplates.get_translations(interactive_template, language_id),
+           InteractiveTemplates.get_translations(interactive_template, language_id)
+           |> InteractiveTemplates.get_clean_interactive_content(
```
Comment (pankaj-ag) [inline lib/glific/messages.ex:250]: Can't we move this function to get_translations itself?

File: lib/glific_web/schema/interactive_template_types.ex [hunk 2]
```diff
@@ -43,6 +44,9 @@ defmodule GlificWeb.Schema.InteractiveTemplateTypes do
 
     @desc "Match a language id"
     field :language_id, :integer
+
+    @desc "Match a send interactive title field"
+    field :send_interactive_title, :boolean
```
Comment (pankaj-ag) [inline lib/glific_web/schema/interactive_template_types.ex]: send_with_title

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: lib/glific/messages.ex (hunk 1 above)
Comment (AkhileshNegi) [reply lib/glific/messages.ex:250]: @pankaj-ag I thought of the same thing but then the function get_translation definition changes so I intentionally made a different function. Let me know your thoughts on same

File: lib/glific_web/schema/interactive_template_types.ex (hunk 2 above)
Comment (AkhileshNegi) [reply lib/glific_web/schema/interactive_template_types.ex]: @pankaj-ag made requested changes

**Review by pankaj-ag — APPROVED** [review]
Approved.

**Pull request merged**